### PR TITLE
fix: various export transaction history fixes

### PR DESCRIPTION
**Pull request by nicole-obrien**
# Description of change

For exporting transaction history:
- Make headers more verbose
- Separate date and time into their own columns and add `(UTC)` to time column header
- Switch minus sign around so that it is correctly on outgoing transactions
- Add extra column to show formatted value with units

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Update (a change which updates existing functionality)
- Fix (a change which fixes an issue)

## How the change has been tested

Manually tested on MacOS

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that my latest changes pass CI tests
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/shared/lib/transactionHistory.ts [hunk 1]
```diff
@@ -15,26 +18,41 @@ export const generateTransactionHistoryCsvFromAccount = (
     headerParams: ITransactionHistoryHeaderParameters
 ): string => {
     const headerParts = []
-    headerParams.id && headerParts.push('id')
+    headerParams.id && headerParts.push('message id')
```
Comment (cvarley100) [inline packages/shared/lib/transactionHistory.ts:21]: these could be translated

**Review by cvarley100 — CHANGES_REQUESTED** [review]
When you export (at least on Mac) the file system window reopens after every export. It should close and the modal close too. Also not sure why the file name includes "wallet 1", the wallet is called "gaps".

Screenshot 2021-12-14 at 03 28 07

File: packages/shared/lib/transactionHistory.ts (hunk 1 above)
Comment (nicole-obrien) [reply packages/shared/lib/transactionHistory.ts:21]: Will do that

**Comment by nicole-obrien** [comment]
> When you export (at least on Mac) the file system window reopens after every export. It should close and the modal close too. Also not sure why the file name includes "wallet 1", the wallet is called "gaps".

It's because its allowing you to save a file for each account consecutively. This was just for the MVP and I have item on backlog to improve this, i.e. allowing them to select which account to export.

Would also be an improvement to add this to the menu on a specific account too, below where we view address history we could have a button to export this specific account.

As for the name of the file I will double check this..

**Comment by nicole-obrien** [comment]
@cvarley100 

I have moved the export to the AccountActions menu and it is now on a per account basis. I have also localised the field names for the csv file.

It's now ready for a rereview 😊

**Review by cvarley100 — APPROVED** [review]

**Pull request merged**